tests/test_shift.py

```diff
@@ -1,17 +1,18 @@
 import unittest
 import pytest
 
-from code import DistributionShift
 import numpy as np
 import pandas as pd
 
-from sklearn.datasets import make_classification
-import random
+from distributions import DistributionShift
 
 
 class TestDistributionShift(unittest.TestCase):
     def setUp(self):
```

Comment:
I am not familiar with this method. Will this automatically be called on testing? Otherwise `X`, `X_array` and `y` could just be set up as class fixtures.